src/components/tooltip/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { FC, ReactNode, useState, useRef } from "react"
+
+import PopUp from "./popUp"
+
+export enum TooltipPosition {
+  top = "top",
+  topLeft = "topLeft",
+  topRight = "topRight",
+  bottom = "bottom",
+  bottomLeft = "bottomLeft",
+  bottomRight = "bottomRight",
+  left = "left",
+  right = "right",
+}
+
+const spacingPosition = {
+  top: "top",
+  topLeft: "top",
```

Comment:
That's an indicator on the first comment I made about the position options Like at first I did not understand what the intention is behind this...